pom.xml

```diff
@@ -85,7 +85,7 @@ LICENSE file.
     <mongodb.version>3.0.3</mongodb.version>
     <mongodb.async.version>2.0.1</mongodb.async.version>
     <orientdb.version>2.1.8</orientdb.version>
-    <redis.version>2.0.0</redis.version>
+    <jedis.version>2.8.1</jedis.version>
```

Comment:
does the jedis library version still work with the same range of redis versions? It's fine if it doesn't, so long as folks are unlikely to want to test the versions that are dropped and we add a release note.